_i18n/pt_BR/factors/dependencies.md

```diff
@@ -0,0 +1,55 @@
+**Idealmente, um aplicativo iOS-factor nunca depende da existência implícita de pacotes a nível de sistema**. Ela declara todas dependências, completamente e exatamente através de um manifesto de declaração de dependências. Isso inclui as versões exatas do [Xcode](https://developer.apple.com/xcode), [CocoaPods](https://cocoapods.org) e [fastlane](https://fastlane.tools).
+
+O benefício da declaração explícita de dependências é que isso simplifica a configuração inicial do projeto para novos desenvolvedores, assim como ajuda a ter um sistema de build confiável que também é capaz de executar builds antigos novamente de uma forma que seja reproduzível. Um novo desenvolvedor pode baixar o código fonte do aplicativo na sua máquina, tendo somente o _runtime_ da linguagem e o gerenciador de dependências instalados como os únicos pré-requisitos.
+
+**Ao especificar as dependências exatas, você consegue regerar um build de 6 meses atrás, sabendo que vai dar certo** dado que vai usar a mesma versão do Xcode, CocoaPods e do Swift.
```

Comment:
I'd change here `sabendo que vai dar certo** dado que vai usar a mesma versão do Xcode, CocoaPods e do Swift.` to `sabendo que irá funcionar dado que serão utilizadas as mesmas versões do Xcode, CocoaPods e do Swift.`